lib/src/data/data-layer.ts

```diff
@@ -78,12 +78,12 @@ export class DataLayer<T> {
         return userID ? `${ store }-${ this._id }-${ userID }` : `${ store }-${ this._id }`;
     }
 
-    protected isStorageAvailable(store: Storage): boolean {
+    protected isLocalStorageAvailable(): boolean {
         try {
             const testValue:string  = "test";
```

Comment:
Shall we use a very specific value?

```suggestion
            const testValue:string  = "__ASGARDEO_AUTH_CORE_LOCAL_STORAGE_TEST__";
```

Reply:
Fixed with cea03c9e92b66a8f605aa23ddb088611c16f7a80